### PR TITLE
Release 4.14.0

### PR DESCRIPTION
**Pull request by johnhwhite**
- Add `skyThemeIf` and `skyThemeClass` directives #204

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.14.0 (2021-03-08)
+
+- Add `skyThemeIf` and `skyThemeClass` directives [#204](https://github.com/blackbaud/skyux-theme/pull/204)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md:3]: Missing closing period `.`. Also, should this be `Added` to be consistent with our past tense?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.14.0](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.14.0)

File: CHANGELOG.md (hunk 1 above)
Comment (blackbaud-paulah) [inline CHANGELOG.md:3]:
```suggestion
- Added `skyThemeIf` and `skyThemeClass` directives [#204](https://github.com/blackbaud/skyux-theme/pull/204)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.14.0](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.14.0)

**Review by blackbaud-paulah — APPROVED** [review]

**Pull request merged**